config/environments/production.rb

```diff
@@ -52,7 +52,12 @@
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   config.force_ssl = true
-  config.ssl_options = { hsts: false }
+  config.ssl_options = {
+    hsts: false,
+    redirect: {
+      exclude: ->(request) { request.path.start_with?('internal') }
```

Comment:
you’re inconsistent with the next file between `/internal` and `internal`

Reply:
Ugh, thanks. I tried multiple things and forgot to sync after the last iteration.